src/components/ContinuationIn/ManualBusinessInfo.vue

```diff
@@ -227,8 +232,9 @@ import { Component, Emit, Mixins, Watch } from 'vue-property-decorator'
 import { Action, Getter } from 'pinia-class'
 import { mask } from 'vue-the-mask'
 import { useStore } from '@/store/store'
-import { DateMixin } from '@/mixins'
+import { DateMixin, DocumentMixin } from '@/mixins'
 import { ExistingBusinessInfoIF } from '@/interfaces'
+import { PdfPageSize } from '@/enums'
```

Comment:
I think you don't need DocumentMixin or PdfPageSize in here.

Reply:
I think you don't need getKeycloakGuid either.

Reply:
Or customErrorMessage.

Reply:
Removed unused codes